packages/react-components/src/organisms/__tests__/TeamCreateOutputContributorCard.test.tsx

```diff
@@ -5,73 +5,71 @@ import { ComponentProps } from 'react';
-          ]),
-        ),
-      );
+describe('Labs', () => {
+  it('should render lab placeholder when no value is selected', () => {
+    const { getByText } = render(
+      <TeamCreateOutputContributorsCard {...props} />,
+    );
+    expect(getByText(/start typing/i)).toBeVisible();
+  });
+  it('should render provided values', () => {
+    const { getByText } = render(
+      <TeamCreateOutputContributorsCard
+        {...props}
+        labs={[
+          { label: 'One Lab', value: '1' },
+          { label: 'Two Lab', value: '2' },
+        ]}
+      />,
+    );
+    expect(getByText(/one lab/i)).toBeVisible();
+    expect(getByText(/two lab/i)).toBeVisible();
+  });
+  it('should be able to select from the list of options', async () => {
+    const loadOptions = jest.fn();
+    loadOptions.mockReturnValue(
+      new Promise((resolve) =>
+        resolve([
+          { label: 'One Lab', value: '1' },
+          { label: 'Two Lab', value: '2' },
+        ]),
+      ),
+    );
 
-      const { getByText, getByLabelText, queryByText } = render(
-        <TeamCreateOutputContributorsCard
-          {...props}
-          loadOptions={loadOptions}
-        />,
-      );
-      userEvent.click(getByLabelText(/Labs/i));
-      await waitFor(() =>
-        expect(queryByText(/loading/i)).not.toBeInTheDocument(),
-      );
-      userEvent.click(getByText('One Lab'));
-      expect(props.onChange).toHaveBeenCalledWith([
-        { label: 'One Lab', value: '1' },
-      ]);
-    });
-    it('should render message when there is no match', async () => {
-      const loadOptions = jest.fn();
-      loadOptions.mockReturnValue(new Promise((resolve) => resolve([])));
-      const { getByLabelText, queryByText } = render(
-        <TeamCreateOutputContributorsCard
-          {...props}
-          loadOptions={loadOptions}
-        />,
-      );
-      userEvent.click(getByLabelText(/Labs/i));
-      await waitFor(() =>
-        expect(queryByText(/loading/i)).not.toBeInTheDocument(),
-      );
-      expect(queryByText(/no labs match/i)).toBeInTheDocument();
-    });
+    const { getByText, getByLabelText, queryByText } = render(
+      <TeamCreateOutputContributorsCard
+        {...props}
+        labSuggestions={loadOptions}
+      />,
+    );
+    userEvent.click(getByLabelText(/Labs/i));
+    await waitFor(() =>
+      expect(queryByText(/loading/i)).not.toBeInTheDocument(),
+    );
+    userEvent.click(getByText('One Lab'));
+    expect(props.onChangeLabs).toHaveBeenCalledWith([
+      { label: 'One Lab', value: '1' },
+    ]);
+  });
+  it('should render message when there is no match', async () => {
+    const loadOptions = jest.fn();
+    loadOptions.mockReturnValue(new Promise((resolve) => resolve([])));
+    const { getByLabelText, queryByText } = render(
+      <TeamCreateOutputContributorsCard
+        {...props}
+        labSuggestions={loadOptions}
+      />,
+    );
+    userEvent.click(getByLabelText(/Labs/i));
+    await waitFor(() =>
```

Comment:
Is it possible to use the waitForElementToBeRemoved method here? Is there any advantage in doing so?